packages/react-router/docs/api/Route.md

```diff
@@ -56,7 +56,7 @@ All three [render methods](#route-render-methods) will be passed the same three
 - [location](./location.md)
 - [history](./history.md)
 
-## component
+## component: component | func
```

Comment:
You're right, but in the render prop is specified `func`, so I thought it was a good idea to add it.